pkg/scripts/common.sh

```diff
@@ -0,0 +1,12 @@
+# Prints message to stdout
+# params: $1 - message, $2 - log level
+log(){
+  if [ "$#" -eq 0 ]; then
+    echo "Usage: log <message>"
+    return 1
+  elif [ "$#" -eq 2 ]; then
+    echo "$(date +"%Y-%m-%d %H:%M:%S") - [$2] - $1"
```

Comment:
<minor> I suggest we drop dashes and go with the the following pattern for log report:

`<date+time> [level] <message>`